packages/ckeditor5-modern/src/services/httpClient.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import { getAccessToken } from '../utils/tokenStorage';
+
+const commonHeaders = {
+  Accept: 'application/json, application/xml, text/play, text/html, *.*',
+  'Content-Type': 'application/json',
+  'Access-Control-Allow-Origin': '*',
+};
+axios.defaults.headers.common = commonHeaders;
+
+export const HOVHttpClient = (allowToken: boolean = true) => {
+  const hovHttpClient = axios.create();
+  hovHttpClient.interceptors.request.use(
+    config => {
+      if (getAccessToken() && allowToken) {
+        config.headers.Authorization = `Bearer ${getAccessToken()}`;
+      }
+      return config;
+    },
+    error => Promise.reject(error)
+  );
+  return hovHttpClient;
+};
```

Comment:
Nice. just naming convension. `hovHttpClient`. Also I think we don't need the `allowToken` argument.

Reply:
@vmc08 allowed token is setup because the upload file api is not accepting bearer tokens so thats why it has that condition